libkbfs/kbfs_ops.go

```diff
@@ -426,10 +417,20 @@ func (fs *KBFSOpsStandard) createAndStoreTlfIDIfNeeded(
 		return err
 	}
 
+	matches, epoch, err := h.tlfID.GetEpochFromTeamTLF(teamID)
+	if err != nil {
+		return err
+	}
+	if matches {
+		epoch++
+	} else {
+		epoch = 0
```

Comment:
If it doesn't match, does it mean the TLF is not migrated to impl yet?

Reply:
Yeah, exactly.